maps_dashboards/server/saved_objects/map_saved_object.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const mapSavedObjectsType: SavedObjectsType = {
+  name: 'map',
+  hidden: false,
+  namespaceType: 'agnostic',
+  management: {
+    defaultSearchField: 'title',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return obj.attributes.title;
+    },
+    getInAppUrl(obj) {
+      return {
+        path: `/app/maps-dashboards#/${encodeURIComponent(obj.id)}`,
+        uiCapabilitiesPath: 'map.show',
+      };
+    },
+    getEditUrl(obj) {
+      return `/management/opensearch-dashboards/objects/map/${encodeURIComponent(obj.id)}`;
+    },
+  },
+  mappings: {
+    properties: {
+      title: { type: 'text' },
+      description: { type: 'text', index: false },
```

Comment:
I think you might want the description to be searchable right?

Reply:
will update